config/fake_data_categories/patient_fake_data_categories.json

```diff
@@ -40,12 +40,18 @@
         1
     ],
     "ADMIT_DIV": [
-        "Test1",
-        "Test2"
+        "Medicine",
```

Comment:
Thank you for this. Could we please include a note highlighting that real categories are being used in the fake data generation (and that these are still being randomly assigned) and why test categories couldn't be used as it will be helpful for future readers.